changelog/fix-plugin-upgrade

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Missing file during plugin upgrade
```

Comment:
I suggest to not add any changelog for this PR as the problem this PR is trying to fix did not exist in previous version.

Reply:
Sorry, I meant not to remove the whole changelog file because then GH check for changelog will fail.

You can:
- Run `npm run changelog`.
- Choose `patch`.
- Leave empty when prompted with "Changelog entry. May be left empty if this change is particularly insignificant".
- Leave a comment why you put an empty changelog.

I know that `npm run changelog` isn't too clear.

I talked about it several times, eg p1683308423223529/1683298938.613009-slack-CGGCLBN58

Reply:
Oh I misunderstood. Added an empty changelog with a comment